janitor/functions.py

```diff
@@ -3227,6 +3240,11 @@ def select_columns(
         [str, callable, Pattern, slice, list],
     )
 
+    # applicable for any
+    # list-like object (ndarray, Series, pd.Index, tuple, ...)
+    if pd.api.types.is_list_like(search_column_names):
```

Comment:
Oooh, cool catch! I didn't know the pandas Types API provided this haha.

Reply:
@eric, pandas has some very rich features that makes life easy :); yeah, not really sure; I wouldnt want to tamper too much with someone else's code; I think I'll take this off, and if users are interested in it, maybe include it.

Reply:
Actually, let's not take it off, I think it's a great idea :smile:.